src/main/java/strategy/handlers/PurchaseHandler.java

```diff
@@ -0,0 +1,18 @@
+package strategy.handlers;
+
+import db.Storage;

```

Comment:
This package declaration should be followed by a newline to separate it from the import statements.